src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -143,6 +143,9 @@ public bool SetNullableContext(byte? value)
             }
         }
 
+        private static readonly ObjectPool<PooledDictionary<Symbol, Symbol>> s_duplicateMemberSignatureDictionary =
```

Comment:
>s_duplicateMemberSignatureDictionary [](start = 77, length = 36)

Why do you need a custom pool instead of using `PooledDictionary<Symbol, Symbol>.GetInstance()`?

Reply:
Need a custom comparer